src/modules/tasks/Journal.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect, useContext } from "react";
+import { makeStyles } from "@material-ui/core";
+import AppContext from "../app/AppContext";
+
+
+const useStyles = makeStyles(theme => ({
+  textBox: {
+    fontSize: "1em",
+    width: "100%",
+    border: "none",
+    padding: "1em",
+    borderRadius: "10px",
+    backgroundColor: "rgb(40,44,52, .5)",
+    outline: "none",
+    color: "blanchedalmond",
+  },
+}));
+
+const Journal = ({checkReady}) => {
+  const [state, dispatch] = useContext(AppContext);
+  const { selectedTask } = state;
+
+  const [entry, setEntry] = useState("");
+  // will need this for later logic addition
+  // const [currentTask, setCurrentTask] = useState(selectedTask);
+
+  const classes = useStyles();
+
+  useEffect(()=> {
+    entry ? checkReady(true) : checkReady(false)
+    // we need logic to bundle the updated task object to have completed true/false, and 
```

Comment:
On that note, we'll need to pass the blob from ImageCapture back up to Task as well.